Makefile

```diff
@@ -31,9 +31,6 @@ benchmark:
 errcheck:
 	errcheck -ignoretests -ignoregenerated ./...
 
-start_tracer:
```

Comment:
is there no longer any setup code for the tracer? don't I need something running locally? are there setup steps if we're running this for the first time in a new cluster?

Reply:
No setup - it all goes to the globally public opentracing.windmill.build, regardless of running locally or in whatever cluster.

I've pushed its k8s config to https://github.com/windmilleng/honeycomb-opentracing-proxy, which might not be the best place for it.

Reply:
I should say: I wanted to keep it opt-in, so users aren't unwittingly sending us their data, so `--trace` is now required if you want a trace.